### PR TITLE
Adjust ad data slots for ps and ws, add new attribute and update tests

### PR DESCRIPTION
**Pull request by chris-hinds**
Resolves N/A

**Overall change:**
- Updates data slots to match

```
WS - Live /4817/bbcworldservice.live.site.${service}
PS - Live /4817/bbccom.live.site.amp.news
WS - test /4817/bbccom.test.site.amp.news
PS - test /4817/bbccom.test.site.amp.news
```

- Adds new `data-multi-size-validation="false"` attribute.

- Correct sets asset_type to either `index` or `story` based on page type

**Code changes:**

- _A bullet point list of key code changes that have been made._
- _When describing code changes, try to communicate **how** and **why** you implemented something a specific way, not just **what** has changed._

---

- [ ] I have assigned myself to this PR and the corresponding issues
- [ ] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [ ] I have assigned this PR to the Simorgh project

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false yarn test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

File: src/app/containers/Ad/Amp/AdSlot.jsx [hunk 1]
```diff
@@ -54,10 +71,14 @@ const slotConfigurations = {
   },
 };
 
-const AdSlot = ({ service, slotType }) => {
+export const getAssetType = pageType =>
```
Comment (HarveyPeachey) [inline src/app/containers/Ad/Amp/AdSlot.jsx:74]: Just to double check these are the only asset types which currently have amp ads served?

**Review by HarveyPeachey — APPROVED** [review]
LGTM Just added a question for clarity

File: src/app/containers/Ad/Amp/AdSlot.jsx (hunk 1 above)
Comment (HarveyPeachey) [reply src/app/containers/Ad/Amp/AdSlot.jsx:74]: Just wanted to check because this function will always default to`index`

Comment (chris-hinds) [reply src/app/containers/Ad/Amp/AdSlot.jsx:74]: correct

**Review by fatimaarhab — APPROVED** [review]
lgtm

**Pull request merged**